tests/compat/check.py

```diff
@@ -0,0 +1,48 @@
+import json
+
+f = open('rpc_compat.json')
+compat = json.load(f)
+
+versions = compat['all_versions']
+compat_table = [[True for x in range(len(versions))] for y in range(len(versions))]
+
+for v in versions:
+	print("downloading version ", v, "...")
+	print("downloading corpus for version", v, "...")
+
+for version, not_compat_list in compat['not_compatbile'].items():
+	version_compat_row = compat_table[versions.index(version)]
+	for not_compat in not_compat_list:
+		if not_compat[0] == '>':
+			until = versions.index(not_compat[1:].strip()) + 1
+			for i in range(until, len(versions)):
+				version_compat_row[i] = False
+		elif not_compat[0] == '<':
+			starting = versions.index(not_compat[1:].strip())
+			for i in range(0, starting):
+				version_compat_row[i] = False
+		elif not_compat[0] == '+':
+			add = versions.index(not_compat[1:].strip())
+			version_compat_row[add] = True
+		else:
+			print("unkown indicator ", not_compat)
+
+for i, compat in enumerate(compat_table):
```

Comment:
do you think that this `check.py` will ultimately live in a separate repository with collected artifacts or remain in the redpanda repo?